extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -56,6 +62,30 @@ def get(self, objectId: str = None):
                 application/json:
                   schema: Error
         """
+        single_user_group = (
+            DBSession().query(Group).filter(Group.name == self.current_user.username).first()
```

Comment:
Is this query indexed?

Reply:
No. Do you think it will significantly affect perf? We will have a ~hundred groups (including single-user groups).
If I add `index=True` to the corresponding column in SA, will it require db migration?

Reply:
Worth doing now while we can still mess with the DB.

Reply:
Done in [#906](https://github.com/skyportal/skyportal/pull/906) on SP.